code/ui/blocks/src/blocks/Source.stories.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Source } from './Source';
+import * as ButtonStories from '../examples/Button.stories';
+import { SourceContext } from './SourceContainer';
+
+const meta: Meta<typeof Source> = {
+  component: Source,
+  parameters: {
+    relativeCsfPaths: ['../examples/Button.stories', '../blocks/Source.stories'],
+  },
+  decorators: [
+    (Story, { parameters: { sources = {} } }) => (
+      <SourceContext.Provider value={{ sources }}>
+        <Story />
+      </SourceContext.Provider>
+    ),
+  ],
+};
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const DefaultAttached = {};
+
+// Still working on this
+// export const Of: Story = {
+//   args: {
+//     of: ButtonStories.Primary,
+//   },
+//   parameters: {
+//     sources: {
+//       'storybook-blocks-example-button--primary': `
+//         const emitted = 'source';
+//       `,
+//     },
+//   },
+// };
```

Comment:
Just need to figure out if the `docs.source.type` and `docs.source.transformSource` parameters need to be documented, and if props need to be added for them.